src/cleanandgo/Update.java

```diff
@@ -46,13 +47,13 @@ private static void insert() {
 				addNewEquipment();
 				break;
 			case '2':
-				// addNewService();
+				 addNewService();
```

Comment:
When you will move addNewService to the Service.java, then you can call it from here as Service.addNewService();
P.S. Make sure that the method is declared as 'public static'